pkgs/applications/misc/1password-gui/beta.nix

```diff
@@ -44,12 +44,19 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "1password";
-  version = "8.8.0-165.BETA";
+  version = "8.8.0-183.BETA";
 
-  src = fetchurl {
-    url = "https://downloads.1password.com/linux/tar/beta/x86_64/1password-${version}.x64.tar.gz";
-    sha256 = "sha256-ZpKAkuIMeHEFdz/od/sKDh8VXoWOZYO8GjvMiho3D4A=";
-  };
+  src =
+    if stdenv.hostPlatform.system == "aarch64-linux" then
+      fetchurl {
+        url = "https://downloads.1password.com/linux/tar/beta/aarch64/1password-${version}.arm64.tar.gz";
+        sha256 = "sha256-LqZAewI6dU9lalVL6A/oPUopS8HZ09SZpRrurVq2wKI=";
+      }
+    else
```

Comment:
while you're at it, maybe support macOS downloads as well?

Reply:
I don't think making this very Linux specific derivation mac compatible would be in scope for a version bump.